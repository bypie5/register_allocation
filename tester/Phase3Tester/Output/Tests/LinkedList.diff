--- SelfTestCases/LinkedList.out	2020-03-05 22:31:29.000000000 -0800
+++ ./Output/Tests/LinkedList.out	2020-03-08 22:09:00.000000000 -0700
@@ -14,13 +14,8 @@
 39
 25
 2220000
--555
--555
-28
-22
-25
-33300000
-22
-25
-44440000
-0
+--- Error (ProgramFault) ---
+invalid argument types: (integer1, data address<Heap.6>+0)
+  Main() line 36
+  LL.Start() line 763
+  List.Delete() line 307
