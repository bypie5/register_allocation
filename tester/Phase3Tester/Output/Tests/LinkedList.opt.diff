--- SelfTestCases/LinkedList.opt.out	2020-03-05 22:31:29.000000000 -0800
+++ ./Output/Tests/LinkedList.opt.out	2020-03-09 14:43:12.000000000 -0700
@@ -7,20 +7,8 @@
 39
 25
 1
-0
-10000000
-28
-22
-39
-25
-2220000
--555
--555
-28
-22
-25
-33300000
-22
-25
-44440000
-0
+--- Error (ProgramFault) ---
+require data address, found integer1
+  Main() line 11
+  LL.Start() line 626
+  List.Search() line 368
