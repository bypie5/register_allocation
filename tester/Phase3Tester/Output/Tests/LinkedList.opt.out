25
10000000
39
25
10000000
22
39
25
1
--- Error (ProgramFault) ---
require data address, found integer1
  Main() line 11
  LL.Start() line 626
  List.Search() line 368
